doc/source/myrocks/install.rst

```diff
@@ -29,11 +29,11 @@ It is recommended to install Percona software from official repositories:
 
    * For Debian or Ubuntu::
 
-      $ sudo apt-get install percona-server-rocksdb-8.0
+      $ sudo apt-get install percona-server-rocksdb
 
    * For RHEL or CentOS::
 
-      $ sudo yum install Percona-Server-rocksdb-80.x86_64
+      $ sudo yum install percona-server-rocksdb-x86_64
```

Comment:
I don't think this exists: `percona-server-rocksdb-x86_64`, maybe it would be `percona-server-rocksdb.x86_64` or just `percona-server-rocksdb`